tests/test_new_client.py

```diff
@@ -176,3 +177,68 @@ def test_client_anthropic_response():
     )
     assert user.name == "Jason"
     assert user.age == 10
+
+
+def test_client_anthropic_bedrock_response():
```

Comment:
can we mark this test as skipped since i dont have permissions

Reply:
Done